starter-code/src/clue.js

```diff
@@ -1,94 +1,224 @@
+  age: 36,
+  image: 'https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg',
+  occupation: 'Socialité'
+};
+
+var mrMustard = {
+  name: 'mrMustard',
+  first_name: 'Jack',
+  last_name: 'Mustard',
+  color: 'yellow',
+  description: 'He is a former football player who tries to get by on his former glory',
+  age: 62,
+  image: 'https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg',
+  occupation: 'Retired Football Player'
+};
 
 // Weapons
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
+var rope = { 
+  name: 'rope',
+  weight: 10
+};
+
+var knife = {
+  name: 'knife',
+  weight: 8
+};
+
+var candlestick = {
+  name: 'candlestick',
+  weight: 2
+};
+
+var dumbbell = {
+  name: 'dumbbell',
+  weight: 30
+};
+
+var poison = {
+  name: 'poison',
+  weight: 2
+};
+
+var axe = { 
+  name: 'axe',
+  weight: 15
+};
+
+var bat = {
+  name: 'bat',
+  weight: 13
+};
+
+var trophy = {
+  name: 'trophy',
+  weight: 25
+};
+
+var pistol = {
+  name: 'pistol',
+  weight: 20
+};
 
 // Rooms
-name: Dinning Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
+var dining = {
+  name: 'Dining Room'
+};
+
+var conservatory = {
+  name: 'Conservatory'
+};
+
+var kitchen = {
+  name: 'Kitchen'
+};
+
+var study ={
+  name: 'Study'
+};
+
+var library = {
+  name: 'Library'
+};
+
+var billiard = {
+  name: 'Billiard Room'
+};
+
+var lounge = {
+  name: 'Lounge'
+};
+
+var ballroom = {
+  name: 'Ballroom'
+};
+
+var hall = {
+  name: 'Hall'
+};
+
+var spa = {
+  name: 'Spa'
+};
+
+var living = {
+  name: 'Living Room'
+};
+
+var observ = {
+  name: 'Observatory'
+};
+
+var theater = {
+  name: 'Theater'
+};
+
+var guest = {
+  name: 'Guest House'
+};
+
+var patio = {
+  name: 'Patio'
+};
+
+
 
 // Characters Collection
 var charactersArray = [];
 
+charactersArray.push(mrGreen, drOrchid, profPlum, missScarlet, mrsPeacock, mrMustard);
+
 // Rooms' Collection
 var roomsArray = [];
 
+roomsArray.push(dining, conservatory, kitchen, study, library, billiard, lounge, ballroom, hall, spa, living, observ, theater, guest, patio);
+
 // Weapons Collection
 var weaponsArray = [];
+
+weaponsArray.push(rope, knife, candlestick, poison, axe, bat, trophy, pistol);
+
+// Random Selector 
+randomSelector = function (array) {
+  var random = Math.floor(Math.random()*array.length);
+  return array[random];
+};
+
+// Mystery
+
+var mysteryEnvelope = [];
+
+pickMistery = function (killer, weapon, room) {
+  var randomKillerFirst = randomSelector(killer).first_name;
+  var randomKillerLast = randomSelector(killer).last_name;
```

Comment:
This will cause the first name to not match the second name. The idea is to get one element from the array and then read both values